ui/ui-frontend/projects/referential/src/app/logbook-management-operation/logbook-management-operation-preview/logbook-management-operation-preview.component.html

```diff
@@ -107,3 +121,39 @@
     <button matDialogClose class="btn cancel link">{{ 'COMMON.CANCEL' | translate }}</button>
   </mat-dialog-actions>
 </ng-template>
+
+<ng-template #confirmCancelOperationDialog>
+  <mat-dialog-content>
+    <label> {{ 'LOGBOOK_MANAGEMENT_OPERATION_ACTIONS.POP_UP_' + actionId + '_TITLE' | translate }} </label>
+    <p
+      class="text bold"
+      [innerHTML]="
+        'LOGBOOK_MANAGEMENT_OPERATION_ACTIONS.POP_UP_' +
+          actionId +
+          '_' +
+          (operation?.stepCancellable ? 'FORCED' : 'NON_FORCED') +
+          '_MODE_MESSAGE' | translate
+      "
+    ></p>
+    <div *ngIf="operation?.stepCancellable">
+      <label>
+        {{ 'LOGBOOK_MANAGEMENT_OPERATION_ACTIONS.POP_UP_' + actionId + '_FORCED_MODE_REASON' | translate }}
+      </label>
+      <vitamui-common-textarea
+        class="col"
+        [formControl]="reason"
+        [maxlength]="500"
+        [minlength]="3"
+        [required]="true"
+        [placeholder]="'LOGBOOK_MANAGEMENT_OPERATION_ACTIONS.POP_UP_CANCEL_FORCED_MODE_REASON_PLACEHOLDER' | translate"
+      >
+      </vitamui-common-textarea>
```

Comment:
label => p pour donner de l'espacement vertical (et un label sans "for" n'est pas utile).

col => w-100 pour virer les paddings et prendre toute la largeur

Supprimer les min et max length en double par rapport au validators (on garde required pour avoir l'"*", même si normalement on ne devrait pas avoir besoin de le préciser, mais c'est parce que le composant textarea et vieux et mal foutu).

```suggestion
      <p>
        {{ 'LOGBOOK_MANAGEMENT_OPERATION_ACTIONS.POP_UP_' + actionId + '_FORCED_MODE_REASON' | translate }}
      </p>
      <vitamui-common-textarea
        class="w-100"
        [formControl]="reason"
        [required]="true"
        [placeholder]="'LOGBOOK_MANAGEMENT_OPERATION_ACTIONS.POP_UP_CANCEL_FORCED_MODE_REASON_PLACEHOLDER' | translate"
      >
      </vitamui-common-textarea>
```